home.html

```diff
@@ -26,9 +26,12 @@
                </ul>
             </div>
 			 
-            <center>
-               <p>navigation bar/burger nav</p>
-            </center>
+            <div class = "logo">
```

Comment:
I have added a 'logo' placeholder which will eventually be changed to the music database's name.